### DISCUSSION TITLE
HTTP3 Use proxy to send request

### DISCUSSION THREAD

**Discussion by biplovdahal** [aiortc/aioquic#248] [Q&A]
Using the following example

https://github.com/aiortc/aioquic/blob/main/examples/http3_client.py

Currently running it like this 

python3 examples/http3_client.py 'https://www.google.com/'

I want to send this request behind a proxy. How can I do so?

**Comment by jlaine** [comment]
I have no idea, are there even any proxies which support HTTP/3?
